discovery-provider/plugins/notifications/src/email/notifications/index.ts

```diff
@@ -48,15 +49,17 @@ export const getUsersCanNotifyQuery = async (
   frequency: EmailFrequency,
   pageCount: number,
   lastUser: number
-) =>
-  await identityDb
+) => {
+  const userIds = Array.from({ length: pageCount }, (_, index) => index + lastUser + 1)
```

Comment:
not sure if this helps considering user IDs are not sequential

Reply:
yeah you're right, limiting this to match the query down below